app/src/main/java/com/droidhats/campuscompass/roomdb/AppDB.kt

```diff
@@ -0,0 +1,34 @@
+package com.droidhats.campuscompass.roomdb
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+@Database(entities = [ShuttleBus_Loyola_Entity::class,ShuttleBus_SGW_Entity::class],version = 3, exportSchema = false)
+abstract class AppDB : RoomDatabase() {
+
+   //When we invoke this function it will call the DAO class
+    abstract fun shuttleBusDAO(): ShuttleBus_DAO
+
+    companion object {
+        private var INSTANCE: AppDB? = null
+        fun getInstance(context: Context): AppDB? {
+            if (INSTANCE == null) {
```

Comment:
It's nice that we use singleton here, I think it's essential. However, despite the logic being the same, we should use a consistent way for declaring our singletons, unless there is a special reason for that. Please see MapRepository and CalendarRepository for references to other already implemented singletons

Reply:
Fixed.